### PR TITLE
Add ICC and WCA vocabularies

### PR DESCRIPTION
**Pull request by cmutel**

### CODE REVIEW

File: dds_glossary/services.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ class GlossaryController:
             ),
         ),
         Dataset(name="ISIC4.rdf", url=fao_url + "ISICRev4/ISIC4.rdf"),
+        Dataset(name="ISIC4.rdf", url=fao_url + "ICCv1.1/ICC11.rdf"),
```
Comment (sami-m-g) [inline dds_glossary/services.py:84]:
```suggestion
        Dataset(name="ICC11.rdf", url=fao_url + "ICCv1.1/ICC11.rdf"),

File: dds_glossary/services.py [hunk 2]
```diff
@@ -81,6 +81,8 @@ class GlossaryController:
...
+        Dataset(name="ISIC4.rdf", url=fao_url + "WCA2020Crops/WCACROPS-core.rdf"),
```
Comment (sami-m-g) [inline dds_glossary/services.py:85]:
```suggestion
        Dataset(name="WCACROPS-core.rdf", url=fao_url + "WCA2020Crops/WCACROPS-core.rdf"),

**Review by sami-m-g — CHANGES_REQUESTED** [review]

**Comment by cmutel** [comment]
Sorry, still waking up

File: dds_glossary/services.py [hunk 3]
```diff
@@ -81,6 +81,8 @@ class GlossaryController:
             ),
         ),
         Dataset(name="ISIC4.rdf", url=fao_url + "ISICRev4/ISIC4.rdf"),
+        Dataset(name="ISIC4.rdf", url=fao_url + "ICCv1.1/ICC11.rdf"),
+        Dataset(name="WCACROPS-core.rdf", url=fao_url + "WCA2020Crops/WCACROPS-core.rdf"),
```
Comment (sami-m-g) [inline dds_glossary/services.py:85]:
```suggestion
        Dataset(name="WCACROPS-core.rdf", url=fao_url + "WCA2020Crops/WCACROPS.rdf"),

**Review by sami-m-g — APPROVED** [review]

**Pull request merged**